src/Blocks/WorldInterface.h

```diff
@@ -65,7 +65,7 @@ class cWorldInterface
 
 	virtual void SetTimeOfDay(int a_TimeOfDay) = 0;
 
-	/** Returns true if it is raining, stormy or snowing at the specified location. This takes into account biomes. */
+	/** Returns true if it is raining or storming at the specified location. This takes into account biomes. */
```

Comment:
This makes an inconsistent API. `IsWeatherWet` considers snow wet, `IsWeatherWetAt` doesn't.
I think a new set of API functions for the rain-only condition should be made instead. There's little difference between rain and storm (still rain), so it'd make sense to use `IsWeatherRaining___` style for both of these, and keep the `__Wet__` to include the snow.

Reply:
I had looked through the code that uses IsWeatherWetAt, and all of those places only want to know about rain or storm. There isn't anywhere in the code that would want to know if there was any precipitation (including snow) so unless you think such a function would be useful for plugins, I don't see enough reason to change it.

Reply:
I actually created the function exactly for the purpose of unifying rain and snow into a single term "wet". If those parts of the code are not interested in snow, they are buggy. I still think the function should be kept with its meaning of "rain or snow".

Reply:
I think my issue is that 'wet' weather in minecraft terms means storm or rain but not snow, as snow does not do many of the things that rain or storm do (put out fire, fill cauldrons). 

What could I call a new function that just takes into account 'minecraft wet' weather, when IsWeatherWet and IsWeatherRain are both taken?

Reply:
Oh, I might have made a mistake. `Wet` is really supposed to be only rain / rain in thunderstorm, not snow. For the rain + snow version, there's `Downfall`. Sorry for the confusion.

So yes, use `Wet` and change it to exclude snow (in all instances), document it, too.